q2cli/core/config.py

```diff
@@ -0,0 +1,100 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2019, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import os
+import configparser
+
+import click
+
+import q2cli.util
+
+
+class cli_config():
+    path = os.path.join(q2cli.util.get_app_dir(), 'cli-colors.theme')
+    valid_types = ['options', 'type', 'default_arg', 'command', 'emphasis',
+                   'problem', 'errors', 'required', 'success']
+    valid_stylings = ['fg', 'bg', 'bold', 'dim', 'underline', 'blink',
+                      'reverse']
+    valid_colors = ['black', 'red', 'green', 'yellow', 'blue', 'magenta',
+                    'cyan', 'white', 'bright_black', 'bright_red',
+                    'bright_green', 'bright_yellow', 'bright_blue',
+                    'bright_magenta', 'bright_cyan', 'bright_white']
+    valid_booleans = {'true': True,
+                      'false': False,
+                      't': True,
+                      'f': False}
+    styles = {'options': {'fg': 'blue'},
```

Comment:
This should probably be an instance variable (initialized in `__init__`) as otherwise you have to think about scope resolution between instance vs class (e.g. what does `self.styles` actually point to?), which gets strange in certain edge-cases. Also, conceptually this represents state whereas the other class-elements really aren't state, just configuration/constants.